mlir/include/mlir/Dialect/SCF/Transforms/TileUsingInterface.h

```diff
@@ -115,13 +108,10 @@ struct SCFTilingOptions {
     return *this;
   }
 
-  /// Specify mapping of loops to devices. This is only respected when the loop
-  /// constructs support such a mapping (like `scf.forall`). Will be ignored
-  /// when using loop constructs that dont support such a mapping (like
-  /// `scf.for`)
-  SmallVector<Attribute> mappingVector = {};
-  SCFTilingOptions &setMapping(ArrayRef<Attribute> mapping) {
-    mappingVector = llvm::to_vector(mapping);
+  /// Specify the reduction dimensions to be tiled.
+  SetVector<unsigned> reductionDims;
+  SCFTilingOptions &setReductionDims(ArrayRef<unsigned> dims) {
+    reductionDims.insert(dims.begin(), dims.end());
```

Comment:
This needs a bit more documentation. Why do we explicitly specify which reduction dims are tiled? What is the behavior if you don't specify these dimensions?

Reply:
It says here https://github.com/llvm/llvm-project/pull/143467/files#diff-3de157757f965faf44ff1ea880201c88a70b6d0b68f0e10bf6bc9032b2ee7669R1770 . You can use it to specify a subset of reduction dimensions to tile. If you dont specify, then actually nothing happens. Added the documentation for that.